core/src/main/java/org/elasticsearch/search/aggregations/metrics/tophits/TopHitsBuilder.java

```diff
@@ -366,7 +186,12 @@ private SearchSourceBuilder sourceBuilder() {
         return sourceBuilder;
     }
 
-    public HighlightBuilder highlightBuilder() {
+    public BytesReference highlighter() {
```

Comment:
nit: given that this method is unused and we cannot return a HighlightBuilder anymore, at least for now, I wonder if it makes sense to remove it rather that changing the return type